app/scrambler.py

```diff
@@ -0,0 +1,59 @@
+"""Scrambler module."""
+import random
+
+from picture import Picture
+from PIL import Image, ImageDraw
+
+
+class Scrambler:
+    """Scrambler class."""
+
+    def scramble_rows(self, picture: Picture) -> Picture:
+        """Split an Image up into rows an rearranges them."""
+        raise NotImplementedError
+
+    def scramble_grid(self, picture: Picture) -> Picture:
+        """Split an Image up into tiles an rearranges them."""
```

Comment:
You seem to be using "an" instead of "and" a lot in these docstrings. These docstrings should be a little more descriptive though.